src/index.coffee

```diff
@@ -17,7 +17,7 @@ class Manager
         connection: {
           uri: options.uri
         },
-        exchange: {
+        exchange: _.defaults {}, options.exchange, {
```

Comment:
Some of these should not be overridable, such as 'type: topic'. The name could be changeable but if its intended to work in a network of systems that gets a bit dodgy.